conference.js

```diff
@@ -1020,7 +1020,11 @@ export default {
                     // Rollback the video muted status by using null track
                     return null;
                 })
-                .then(videoTrack => this.useVideoStream(videoTrack));
+                .then(videoTrack => {
+                    logger.debug(`muteVideo: calling useVideoStream for track: ${videoTrack}`);
```

Comment:
Please don't log full tracks, it's incredibly verbose on Mobile. Yes, I know this specific part of the code does not run on mobile, but I'd rather us not have the habit. What are we interested in, about the track?

Reply:
We are interested to see what parameters do we use for the function. 

Out of curiosity, why printing the jitsiTrack is verbose on mobile. I see we have defined `toString` method and it prints something like `LocalTrack[3,video]`. Am I missing something?

Reply:
yes, `JitsiTrack` is okay to be logged as a string

Reply:
Ah sorry for the noise then! Maybe it's the Jingle IQ that I confused it with. We do have some verbose objects being logged. Anyway, carry on! :-)